### PR TITLE
data :  Yashkapure06

### PR DESCRIPTION
**Pull request by Yashkapure06**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2838"></a>

### CODE REVIEW

File: data/Yashkapure06.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name":"Yash Kapure",
+  "type":"Personal",
+  "bio":"As a full-stack web developer, I have a strong foundation in a range of technologies. My expertise spans from front-end development to back-end programming.One of my favorite technologies to work with is React.js, particularly within the MERN stack. I am currently expanding my skillset to include React Native 📱 for mobile development.",
+  "avatar":"https://avatars.githubusercontent.com/u/61585443?v=4",
```
Comment (senali-d) [inline data/Yashkapure06.json:5]: Use short url for avatar.
```suggestion
  "avatar":"https://github.com/Yashkapure06.png",
```

**Review by senali-d — COMMENTED** [review]
@Yashkapure06, Please look after the inline comment.

File: data/Yashkapure06.json (hunk 1 above)
Comment (eddiejaoude) [reply data/Yashkapure06.json:5]: Good suggestion, but this is not a blocker for the pull request

**Review by eddiejaoude — APPROVED** [review]
Great collaboration 👍 

@Yashkapure06 you can make more pull requests to update your profile. @senali-d made a good suggestion for a future improvement

**Pull request merged**